pom.xml

```diff
@@ -33,6 +33,7 @@
         <jmock.version>2.6.0</jmock.version>
         <slf4j.log4j.version>1.7.5</slf4j.log4j.version>
         <agrona.version>0.3.1</agrona.version>
+        <metrics.version>develop-SNAPSHOT</metrics.version>
```

Comment:
Please back out all changes to this file since the plan now is that we should not include metrics in the Gateway distribution, at least for now.